app/views/warehouse_reports/active_veterans/_sort.haml

```diff
@@ -12,5 +12,4 @@
         - else
           - title = "<span class='icon-checkbox-unchecked'></span> #{details[:title]} #{opt[:direction]}".html_safe
         %li
-          - # FIXME replace permit! with a whitelist and remove from brakeman.ignore
-          = link_to title, params.permit!.merge!(direction: opt[:direction], column: opt[:column])
+          = link_to title, params: {direction: opt[:direction], column: opt[:column]}
```

Comment:
I believe this will lose any previously selected filtering, which the merge with params preserved.

Reply:
Sorting is the only control on the active veterans report.